modules/skeletonRetriever/src/main.cpp

```diff
@@ -881,8 +881,8 @@ class Retriever : public RFModule
         rgbdOpts.put("DepthCarrier", "fast_tcp");
 
         if (!rgbdDrv.open(rgbdOpts)) {
-            yError() << "Unable to talk to depthCamera!";
-            return false;
+            yError() << "Unable to talk to depthCamera!";            
+            yWarning() << "Retrieving camera intrinsics from file";
```

Comment:
```suggestion
            yWarning() << "Unable to talk to depthCamera!";
            yWarning() << "Retrieving camera intrinsics from file";
```

Just turning the error into a warning as it's not blocking.
@mfussi66 did you check that in this situation we correctly retrieve the params from the file instead?

Reply:
On a second thought, why we should allow for going on in this case? The module ought to read from file only when instructed an thus launched accordingly.

Reply:
Yes we do retrieve them correctly. 

> On a second thought, why we should allow for going on in this case? The module ought to read from file only when instructed an thus launched accordingly.

That's a good point, I considered the reading from file option a simple fallback when it's not possible to open the RGBD interface to get the camera options. Maybe it's better to add a command line option to enable reading from file (or command line args) ?